lib/actions/appleMusic.js

```diff
@@ -48,10 +48,18 @@ const METADATA_URI_STARTERS = {
 const PARENTS = {
   song: '0004206calbum%3a',
   album: '00020000album%3a',
-  name: '00020000song%3a',
+  name: '00020000song:',
```

Comment:
It was throwing an error and when I compared with what a Favorite showed, the favorite did not have the colon encoded. Didn't seem consistent to me either.